package/Editor/SceneTools/SceneContextMenuActions/SceneContextMenuActions.cs

```diff
@@ -0,0 +1,141 @@
+
+            menu.ShowAsContext();
+        }
+
+        private static void MoveLayerUp()
+        {
+            GameObject selectedLayer = Selection.activeGameObject;
+            int siblingIndex = selectedLayer.transform.GetSiblingIndex();
+            selectedLayer.transform.SetSiblingIndex(siblingIndex + 1);
+        }
+
+        private static void MoveLayerDown()
+        {
+            GameObject selectedLayer = Selection.activeGameObject;
+            int siblingIndex = selectedLayer.transform.GetSiblingIndex();
+            selectedLayer.transform.SetSiblingIndex(siblingIndex - 1);
+        }
+
+        private static void CloneLayer()
+        {
+            GameObject selectedLayer = Selection.activeGameObject;
+            GameObject clonedLayer = UnityEngine.Object.Instantiate(selectedLayer);
+            clonedLayer.name = selectedLayer.name + " (Clone)";
+        }
+
+        private static void DeleteLayer()
+        {
+            GameObject selectedLayer = Selection.activeGameObject;
+            UnityEngine.Object.DestroyImmediate(selectedLayer);
+        }
+
+        private static void FlipHorizontal()
+        {
+            GameObject selectedLayer = Selection.activeGameObject;
+            SpriteRenderer spriteRenderer = selectedLayer.GetComponent<SpriteRenderer>();
+            if (spriteRenderer != null)
+            {
+                spriteRenderer.flipX = !spriteRenderer.flipX;
+            }
+        }
+
+        private static void FlipVertical()
+        {
+            GameObject selectedLayer = Selection.activeGameObject;
+            SpriteRenderer spriteRenderer = selectedLayer.GetComponent<SpriteRenderer>();
+            if (spriteRenderer != null)
+            {
+                spriteRenderer.flipY = !spriteRenderer.flipY;
+            }
+        }
+
+        private static void RemoveBackground()
+        {
+            GameObject selectedLayer = Selection.activeGameObject;
+            SpriteRenderer spriteRenderer = selectedLayer.GetComponent<SpriteRenderer>();
+            if (spriteRenderer != null && spriteRenderer.sprite != null)
+            {
+                Texture2D texture2D = spriteRenderer.sprite.texture;
+
+                // Ensure the texture is readable and uncompressed
+                string texturePath = AssetDatabase.GetAssetPath(texture2D);
+                TextureImporter importer = AssetImporter.GetAtPath(texturePath) as TextureImporter;
+                if (importer != null)
+                {
+                    importer.isReadable = true;
+                    importer.textureCompression = TextureImporterCompression.Uncompressed;
+                    importer.SaveAndReimport();
+                }
+
+                // Convert texture to uncompressed format
+                Texture2D uncompressedTexture = new Texture2D(texture2D.width, texture2D.height, TextureFormat.RGBA32, false);
+                uncompressedTexture.SetPixels(texture2D.GetPixels());
```

Comment:
What about if the sprite selected is not from Scenario ? Default sprite ?

Reply:
![image](https://github.com/scenario-labs/Scenario-Unity/assets/30622829/a907d2da-4d07-4c50-8cf3-02b2c217033c)

Still have this error on default sprite from Unity